src/components/tryout/shorten-link.tsx

```diff
@@ -110,7 +112,24 @@ export function TryOutTab() {
             </div>
           </CardContent>
           <CardFooter>
-            <Button onClick={handleLinkShortenGeneration}>Shorten Link</Button>
+            <Button
+              onClick={handleLinkShortenGeneration}
+              className={`text-sm text-white bg-black hover:text-green-600 hover:bg-green-50 active:scale-95 active:ring-4 active:ring-green-300 transition-transform duration-300 ease-in-out ${
+                loading
+                  ? "bg-green-300 cursor-not-allowed hover:bg-green-300"
+                  : ""
+              }`}
+              // disabled={loading}
```

Comment:
Any reason you commented out this, I thought it will do the job 😅

Reply:
> Any reason you commented out this, I thought it will do the job 😅

I've handled that, no need for it, i'll remove it.

Reply:
> Any reason you commented out this, I thought it will do the job 😅

I've uncommented it